app/controllers/EventsController.scala

```diff
@@ -34,13 +34,16 @@ import scala.util.Try
 
 object EventsController extends EventsController {
   val eventsService: EventsService = EventsService
+  val eventsRepo: EventsMongoRepository = repositories.eventsRepository
```

Comment:
I would say that an EventsController should not know about EventsService and EventsMongoRepository. Please refactor any direct calls on EventsMongoRepository into EventsService.